crates/subspace-bootstrap-node/src/main.rs

```diff
@@ -191,6 +191,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
             info!("Subspace Bootstrap Node started");
 
+            // TODO: spawn this in a dedicated thread
```

Comment:
is this TODO necessary if we are reverting from running exit signals from a dedicated thread ?